src/relay/backend/graph_executor_codegen.cc

```diff
@@ -645,8 +629,8 @@ class GraphExecutorCodegen : public backend::MemoizedExprTranslator<std::vector<
   String mod_name_;
   /*! \brief function metadata */
   Map<String, FunctionInfo> function_metadata_;
-  /*! \brief name map */
-  std::unordered_map<std::string, size_t> name_map_;
+  /*! \brief NameSupply */
+  NameSupply name_supply_ = NameSupply("");
```

Comment:
Maybe you could add a default constructor to `NameSupply`, or give the prefix a default value, so that you don't have to write `= NameSupply("")`, since that occurs in a few places.

Reply:
This contradicts a previous suggestion: https://github.com/apache/tvm/pull/12066#discussion_r929446411

Reply:
Ok, fair enough.